plugins/html-plugin/src/main/java/ru/sbtqa/tag/stepdefs/HtmlSetupSteps.java

```diff
@@ -20,7 +16,6 @@ public static void initHtml() {
     }
 
     private static boolean isNewDriverNeeded() {
-        return Environment.isDriverEmpty()
-                || (!Environment.isDriverEmpty() && !PROPERTIES.getShared());
+        return Environment.isDriverEmpty();
```

Comment:
If it's the only one method inside, maybe it's better to unwrap it and use as is in condition?